checker/bin-devel/wpi-plumelib/README

```diff
@@ -0,0 +1,2 @@
+This directory contains the expected result of type-checking projects from
+https://github.com/plume-lib/, after whole-program inference.
```

Comment:
This file needs to contain instructions for how to produce these files, because changes to plume-lib or to the checkers might cause these tests to fail only because these files need to be regenerated.